.github/workflows/snap-build.yaml

```diff
@@ -0,0 +1,21 @@
+name: Snap Build
+on:
+  pull_request:
+    branches: ["master", "release/**"]
+jobs:
+  build:
+    name: Build the snapd snap
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build the snapd snap
+        uses: snapcore/action-build@v1
+      - name: Rename built snap file
+        run: |
+          sudo mv snapd*.snap snapd.snap
+      - name: Uploading snap artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: snapd.snap
```

Comment:
Could you have used a glob here? I'm not even sure what's here but I think preserving the architecture would be pretty useful.

Reply:
Mmm I didn't try a glob, I can see what happens with a glob easily enough

Reply:
no globs do not work unfortunately, see my etrace test run:

```
2020-07-22T18:47:20.3406383Z Snapped etrace_2a130c0_amd64.snap
2020-07-22T18:47:21.7326601Z ##[group]Run actions/upload-artifact@v1
2020-07-22T18:47:21.7326814Z with:
2020-07-22T18:47:21.7326966Z   name: etrace*.snap
2020-07-22T18:47:21.7327163Z   path: etrace.snap
2020-07-22T18:47:21.7327333Z ##[endgroup]
2020-07-22T18:47:21.9823285Z ##[error]Path does not exist /home/runner/work/etrace/etrace/etrace.snap
2020-07-22T18:47:22.0136894Z ##[error]Exit code 1 returned from process: file name '/home/runner/runners/2.267.1/bin/Runner.PluginHost', arguments 'action "GitHub.Runner.Plugins.Artifact.PublishArtifact, Runner.Plugins"'.
2020-07-22T18:47:22.0191116Z Post job cleanup.
```

Reply:
Maybe it works in v2? Globbing is mentioned in the docs.

Reply:
ah interesting I didn't even realize I was using v2 instead of v1, let me try again with v2 and globbing...

Reply:
it does work :tada:

Reply:
fwiw, `snapcore/action-build` makes the snap file name available as an output parameter.  That was more important back before `upload-artifact@v2` was released.